src/translations/en/service.json

```diff
@@ -18,6 +20,8 @@
   "form.label.json": "JSON",
   "form.select": "Select...",
   "form.serviceName": "Name",
+  "form.executeClientScripts": "Switch JavaScript on",
+  "form.executeClientScripts.more": "If the page you try to load is interactive, rendering it with JavaScript enabled might improve rendering.",
```

Comment:
See changes suggested on the french translations

Reply:
ok done